src/prover/serialization.rs

```diff
@@ -0,0 +1,104 @@
+use ark_ec::pairing::Pairing;
```

Comment:
copied from `mopro-ffi/cricom/serialization.rs` and removed  ethereum related logics.